js/client-mainmenu.js

```diff
@@ -124,14 +124,34 @@
 		// news
 
 		addNews: function () {
-			var newsId = '1990';
-			if (newsId === '' + Dex.prefs('readnews')) return;
-			this.addPseudoPM({
-				title: 'Latest News',
-				html: '<iframe src="/news-embed.php?news' + (window.nodewebkit || document.location.protocol === 'https:' ? '&amp;https' : '') + '" width="270" height="400" border="0" style="border:0;width:100%;height:100%;display:block"></iframe>',
-				attributes: 'data-newsid="' + newsId + '"',
-				cssClass: 'news-embed',
-				height: 400
+			var self = this;
+			$.ajax({
+				url: "https://pokemonshowdown.com/news.json",
+				dataType: "json",
+				success: function (data) {
+					var html = '';
+					for (var i in [0, 1]) {
```

Comment:
Wait, shit, this should be fixed; it works but all the guides online say never to do it.

Reply:
Also it breaks older browsers because it's explicitly incompatible with our polyfill.

Reply:
Oh, my bad.

Reply:
Don't worry, I already fixed it. This was something I was going to fix earlier but forgot.